runtime/runtime.go

```diff
@@ -51,6 +51,9 @@ func init() {
 func parseFailpoints(fps string) (map[string]string, error) {
 	// The format is <FAILPOINT>=<TERMS>[;<FAILPOINT>=<TERMS>]*
 	fpMap := map[string]string{}
+	if len(fps) == 0 {
+		return fpMap, nil
+	}
```

Comment:
Note: Don't think this is needed.

Reply:
If we remove this, When users pass an empty string, then it will return error "bad failpoint xxx".  

If we don't check the fps, then users are supposed to always pass a non-empty string. It seems like not a big deal, so let's keep it as it's for now.